Dnd.Ddd.Model/Character/CharacterStates/Completed.cs

```diff
@@ -20,5 +20,7 @@ internal class Completed : CharacterState
         internal override void SetStrength(Character character, int strength) => throw new InvalidCharacterStateException(character.UiD);
 
         internal override void SetWisdom(Character character, int wisdom) => throw new InvalidCharacterStateException(character.UiD);
+
+        internal override bool CanAdvance(Character character) => false;
```

Comment:
Maybe we could change the name of this prop? It's misleading now

Reply:
Renamed to `CanChangeState`